frigate/http.py

```diff
@@ -1100,7 +1100,13 @@ def latest_frame(camera_name):
         height = int(request.args.get("h", str(frame.shape[0])))
         width = int(height * frame.shape[1] / frame.shape[0])
 
-        frame = cv2.resize(frame, dsize=(width, height), interpolation=cv2.INTER_AREA)
+        try:
+            frame = cv2.resize(
+                frame, dsize=(width, height), interpolation=cv2.INTER_AREA
+            )
+        except Exception as e:
+            logger.debug(f"Failed to resize latest snapshot :: {e}")
+            return "Error resizing snapshot", 500
```

Comment:
I think it's better to check frame, width, and height for invalid values. We should also update the frontend code to prevent the negative values from being passed in the first place.